documents-webapp/src/main/webapp/vue-app/documents/components/body/table/cells/DocumentsFileNameCell.vue

```diff
@@ -101,6 +101,7 @@
   </div>
 </template>
 <script>
+import jsonFileExtension from '../../../../../../../resources/locale/NtFileExtension.json';
```

Comment:
IMO the json file should not be added into locales, you can put it directly into the parent folder or at least inti a json folder under [vue-app/documents/](https://github.com/exoplatform/documents/tree/develop/documents-webapp/src/main/webapp/vue-app)

Reply:
done !